.github/workflows/docker.yaml

```diff
@@ -22,25 +22,16 @@ jobs:
     permissions:
       packages: write
       contents: read
-    runs-on: oxford
+    runs-on: [self-hosted, linux, amd64]
     steps:
       - name: Check out code
-        uses: actions/checkout@v4.1.7
+        uses: actions/checkout@v4
         with:
           fetch-depth: 0
           # Check https://github.com/livepeer/go-livepeer/pull/1891
           # for ref value discussion
           ref: ${{ github.event.pull_request.head.sha }}
 
-      - name: Cleanup hosted runner
```

Comment:
Just checking that this was intentional to remove, since I don't see it re-added anywhere else

Reply:
yeah, moving it to any generic self-hosted runner, where we are not limited by disk space which github runners are (limiting us to i think 20GB disk space).

no longer need to cleanup runner if extra disk space is not needed =)